runner/src/artifacts.rs

```diff
@@ -361,20 +361,20 @@ pub fn list_artifacts(args: &ArtifactArgs) {
     for (name, metadata) in artifacts {
         if args.markdown {
             // Print as markdown
-            println!("## {}\n", name);
+            log::info!("## {}\n", name);
             if let Some(ref desc) = metadata.description {
-                println!("{}", desc);
+                log::info!("{}", desc);
             }
             if let Some(ref url) = metadata.url {
-                println!("- [Download link]({})", url);
+                log::info!("- [Download link]({})", url);
             }
             if let Some(ref repo) = metadata.repo {
-                println!("- [Source repository]({})", repo)
+                log::info!("- [Source repository]({})", repo)
             }
             println!();
         } else {
             // Otherwise simply print the name
-            println!("{}", name)
+            log::info!("{}", name)
```

Comment:
In that one case we actually don't want to use the new logging function,s because it is used to generate the markdown that goes into the website (obviously that won't work anymore with the extra text from `log::info!`). So we need to keep the `println` when `args.markdown == true` in this function.

The page is there for info: https://miralis-firmware.github.io/docs/artifacts